frontend/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>MemoirVerse Sign In</title>
+    <link rel="stylesheet" href="./index.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="welcome-section">
+        <h1>Welcome to MemoirVerse!</h1>
+        <p>Enter your personal details to use all of site features.</p>
+      </div>
```

Comment:
A line space between <div></div> would add readability and legibility

Reply:
Alright! I'll be adding lines between my divs for readablity. Thank you!